s2aff/consts.py

```diff
@@ -15,7 +15,7 @@
 
 PATHS = {
     "ner_training_data": os.path.join(PROJECT_ROOT_PATH, "data", "ner_training_data.pickle"),
-    "ror_data": os.path.join(PROJECT_ROOT_PATH, "data", "v1.1-2022-06-16-ror-data.json"),
+    "ror_data": os.path.join(PROJECT_ROOT_PATH, "data", "v1.18-2023-01-26-ror-data.json"),
```

Comment:
would be a bit cleaner to have a ROR_VERSION variable

Reply:
Good idea, fixed